### PR TITLE
Encrypted volumes creation and import

### PR DESCRIPTION
**Pull request by pchaussalet**

### CODE REVIEW

File: core/service/section/storage-section-service.js [hunk 1]
```diff
@@ -103,25 +144,62 @@ exports.StorageSectionService = AbstractSectionService.specialize({
...
+        }
+    },
+
+    importEncryptedVolume: {
+        value: function(encryptedVolumeImporter) {
+            return this._volumeServices.then(function(volumeServices) {
+                return volumeServices.import(
+                    encryptedVolumeImporter.name,
+                    encryptedVolumeImporter.name,
+                    {},
```
Comment (thibaultzanini) [inline core/service/section/storage-section-service.js:187]: those arguments are right ?

File: ui/sections/storage/inspectors/encrypted-volume-importer.reel/encrypted-volume-importer.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            var self = this;
+            return this._sectionService.listAvailableDisks().then(function(availableDisks) {
+                return self.availableDisks = availableDisks; 
+            });
+        }
+    },
+
+    enterDocument: {
+        value: function(isFirstTime) {
+            this.addRangeAtPathChangeListener("object.disks", this, "_handleDisksChange");
```
Comment (thibaultzanini) [inline ui/sections/storage/inspectors/encrypted-volume-importer.reel/encrypted-volume-importer.js]: isFirstime condition is missing

**Review by thibaultzanini — CHANGES_REQUESTED** [review]

File: core/service/section/storage-section-service.js (hunk 1 above)
Comment (pchaussalet) [reply core/service/section/storage-section-service.js:187]: yes, we need to pass volume name instead of volume GUID for encrypted volumes

**Review by thibaultzanini — COMMENTED** [review]
LGTM

**Pull request merged**